test/unit/utils/rewards.test.ts

```diff
@@ -0,0 +1,34 @@
+import { calculateAnnualizedRewardAmount } from "@interlay/interbtc-api/index";
+import Big from "big.js";
+import { expect } from "chai";
+
+describe("Rewards", () => {
+    describe("calculateAnnualizedRewardAmount", () => {
+        it("should calculate annulized amount correctly", () => {
+            const secsPerYear = 365 * 24 * 60 * 60;
+            // this can and will fail if block times change
+            // (and that's a good thing)
+            const assumedBlockDurationSecs = 12;
+
+            const amountPerBlock = Big(0.042);
+
+            const expectedRewardAmount = amountPerBlock.mul(secsPerYear).div(assumedBlockDurationSecs);
+            const actualRewardAmount = calculateAnnualizedRewardAmount(amountPerBlock, assumedBlockDurationSecs * 1000);
+
+            expect(actualRewardAmount.eq(expectedRewardAmount)).to.eq(
+                true,
+                `Expected result to be equal to ${expectedRewardAmount.toString()} but was: ${actualRewardAmount.toString()}`
+            );
+        });
+
+        it("should throw if blockTimeMs is zero", () => {
+            expect(() => calculateAnnualizedRewardAmount(Big(1), 0)).to.throw("Division by zero");
```

Comment:
Similar here, I suspect we might need an `await` here, so the throw doesn't happen outside of the `describe` block if it occurs. (Not 100% sure tho)

Reply:
This was removed from master, removing from PR.